### ISSUE TITLE
stories index broken

### ISSUE THREAD

**Issue by mmilaprat** [policycompass/policycompass#685]
In stage the stories index doesn't works.

After do a POST to https://services-stage.policycompass.eu/api/v1/searchmanager/rebuildindex
with a Rest Client to try to reindex stories they doesn't appear in the Browse.

The Post return an error:
<h1>Server Error (500)</h1>

**Comment by FabiApfelkern** [comment]
@cueneythizli Can you please check it. It is indeed caused by the Story Manager.

**Comment by cbotsikas** [comment]
The error while rebuilding the index is caused by story manager. It can't list the stories https://services-stage.policycompass.eu/api/v1/storymanager/stories

I've tried to figure it out but, since we have debug disabled (on stage we should have that enabled) and there isn't any proper exception logging inplace, i couldn't.

I've also tried the sqllite db from stage on my local machine with the latest code and worked fine. 

BTW, I've noticed that the api urls of stories, is different in comparison with the rest APIs.
eg:
http://127.0.0.1:8000/api/v1/storymanager/stories/1 <- this doesn't work with 500 error
http://127.0.0.1:8000/api/v1/storymanager/stories?id=1 <- this works
